doc/src/user_guide.rst

```diff
@@ -537,6 +537,31 @@ Example::
     min-fact = 0.3
     max-fact = 2.5
 
+[solver-dual-time-integrator-multip]
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Parameterises multi-p for dual time-stepping with
+
+1. ``pseudo-dt-fact`` --- factor by which the pseudo time-step size
+   changes between multi-p levels:
+
+    *float*
+
+2. ``cycle`` --- nature of a single multi-p cycle:
+
+    ``[(`` *int* ``,`` *int* ``), (`` *int* ``,`` *int* ``), ... (``
+    *int* ``,`` *int* ``)]``
+
+    where the first *int* in each bracketed pair is the multi-p level,
+    and the second *int* in each bracketed pair is the number of pseudo
+    time-steps taken at that multi-p level
```

Comment:
Remark that must start and end on the polynomial order of the simulation.  Also can only change ±1 between levels.